File: nodejs/7 kyu/224.js
// https://www.codewars.com/kata/57faa6ff9610ce181b000028/train/javascript

// Passed

function crap(x, bags, cap) {
  let crapCount = 0;
  for (let row of x) {
    for (let item of row) {
      if (item === "@") {
        crapCount++;
      } else if (item === "D") {
        return "Dog!!";
      }
    }
  }

  let result = crapCount > bags * cap ? "Cr@p" : "Clean";
  return result;
}

const output = crap(
  [
    ["_", "_", "_", "_"],
    ["_", "_", "_", "@"],
    ["_", "_", "@", "_"],
  ],
  2,
  2
);
console.log(output);

File: python/8 kyu/224.py
# https://www.codewars.com/kata/57eaeb9578748ff92a000009/train/python

# Passed

def sum_mix(arr):
    modified_arr = [int(item) for item in arr]
    modified_arr_sum = sum(modified_arr)
    return modified_arr_sum

output = sum_mix([9, 3, '7', '3'])
print(output)